brainglobe_atlasapi/atlas_generation/atlas_scripts/example_mouse.py

```diff
@@ -10,58 +9,135 @@
 
 from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
 
-
-def create_atlas(working_dir, resolution):
-    # Specify information about the atlas:
-    RES_UM = resolution  # 100
-    ATLAS_NAME = "example_mouse"
-    SPECIES = "Mus musculus"
-    ATLAS_LINK = "http://www.brain-map.org"
-    CITATION = "Wang et al 2020, https://doi.org/10.1016/j.cell.2020.04.007"
-    ORIENTATION = "asr"
-
-    # Temporary folder for nrrd files download:
-    download_dir_path = working_dir / "downloading_path"
+# a working example atlas-packaging script, which makes a simplified version
+# of the Allen Mouse Brain Atlas, at 100um resolution. See `template_script.py`
+# for a starting point to package your own atlas.
+__version__ = 0  # This will make the example mouse version 1.0
+ATLAS_NAME = "example_mouse"
+CITATION = "Wang et al 2020, https://doi.org/10.1016/j.cell.2020.04.007"
+SPECIES = "Mus musculus"  # The scientific name of the species,
+ATLAS_LINK = "http://www.brain-map.org"  # The URL for the data files
+ORIENTATION = "asr"  # The orientation of the atlas
+ROOT_ID = 997  # The id of the highest level of the atlas.
+RESOLUTION = 100  # The resolution of your volume in microns.
+
+BG_ROOT_DIR = Path.home() / "brainglobe_workingdir" / ATLAS_NAME
+
+
+def download_resources():
+    """
+    Download the necessary resources for the atlas.
+    """
+    pass
+
+
+def retrieve_reference_and_annotation():
+    """
+    Retrieve the Allen Mouse atlas reference and annotation as two numpy arrays
+    using the allen_sdk.
+
+    Returns:
+        tuple: A tuple containing two numpy arrays. The first array is the
+        reference volume, and the second array is the annotated volume.
+    """
+    # Create temporary download directory
+    download_dir_path = BG_ROOT_DIR / "downloading_path"
+    print(download_dir_path)
     download_dir_path.mkdir(exist_ok=True)
 
-    # Download annotated and template volume:
-    #########################################
+    # Setup the reference space cache
     spacecache = ReferenceSpaceCache(
         manifest=download_dir_path / "manifest.json",
-        # downloaded files are stored relative to here
-        resolution=RES_UM,
+        resolution=RESOLUTION,
         reference_space_key="annotation/ccf_2017",
-        # use the latest version of the CCF
     )
 
-    # Download
-    annotated_volume, _ = spacecache.get_annotation_volume()
-    template_volume, _ = spacecache.get_template_volume()
-    print("Download completed...")
+    reference_volume, _ = spacecache.get_template_volume()
+    annotation_volume, _ = spacecache.get_annotation_volume()
+    expected_reference_hash = (
+        "6c24cae773a5cf256586b0384af0ac93ad68564d211c9bdcff4bee9acf07786a"
+    )
+    expected_annotation_hash = (
+        "451e6a82f531d3db4b58056d024d3e2311703c2adc15cefa75e0268e7f0e69a4"
+    )
+    reference_hash = pooch.file_hash(
```

Comment:
I worry that this is
a) not the way we typically use `pooch` and therefore confusing
b) more importantly, might look scary to new contributors coming more from the biology side

I'm tempted to remove any `pooch`-related things, and mention it in the template script only (which we already do in this PR)?

Reply:
I see your point, the only thing is that pooch here actually serves a purpose, to ensure that the files from the Allen don't change without notifying us. In this instance we would want pooch to confirm the hash of the file. I understand that it's non standard but I think that just comes with using the Allen API in the example. 

An alternative would be to not use the allen API at all, instead just use the files from the [allen informatics archive](download.alleninstitute.org/informatics-archive/current-release/mouse_ccf/). The advantage to this is the example would become more of a standard atlas integration allowing us to use pooch, etc, instead of so much allen specific logic.

Reply:
>An alternative would be to not use the allen API at all, instead just use the files from the [allen informatics archive](https://github.com/brainglobe/brainglobe-atlasapi/pull/download.alleninstitute.org/informatics-archive/current-release/mouse_ccf/).

I think this would be great if you have the patience to implement this? :grin: Would this be possible/easy on the `OntologiesAPI` side too?

The alternative is that we make a new issue to at some point move to a more standard atlas (e.g. we have recently generated a bird template for which we will host the data ourselves, and which has ITK snap annotations - our standard - so we could use that when the preprint is published.)

Reply:
We can tackle this separately with #416 .